### PR TITLE
[Machine-learning][ACR-1755] Add guide for new api

### PR DESCRIPTION
**Pull request by niley7464**
- Add the guide documentation of Machine Learning Inference API
- ACR: ACR-1755

### CODE REVIEW

File: docs/application/native/guides/machine-learning/machine-learning-inference.md [hunk 1]
```diff
@@ -482,6 +482,11 @@ For more information about the pipeline states, see [GStreamer guide](https://gs
     ml_tensors_info_destroy (info);
     ```
 
+    If you want to use tensors which dimension is higher than 4, use the following function.
```
Comment (omar-srbd) [inline docs/application/native/guides/machine-learning/machine-learning-inference.md:485]:
```suggestion
    If you want to use tensors whose dimension is higher than 4, use the following function:
```

**Review by omar-srbd — COMMENTED** [review]
Hello @niley7464, there is a minor change to be made on the changed line.
Please check and incorporate.
Thanks.

File: docs/application/native/guides/machine-learning/machine-learning-inference.md (hunk 1 above)
Comment (niley7464) [reply docs/application/native/guides/machine-learning/machine-learning-inference.md]: Thank you for the suggestion :)
Updated it

**Review by jaeyun-jung — APPROVED** [review]

File: docs/application/native/guides/machine-learning/machine-learning-inference.md [hunk 2]
```diff
@@ -482,6 +482,11 @@ For more information about the pipeline states, see [GStreamer guide](https://gs
     ml_tensors_info_destroy (info);
     ```
 
+    If you want to use tensors whose dimension is higher than 4, use the following function.
```
Comment (omar-srbd) [inline docs/application/native/guides/machine-learning/machine-learning-inference.md:485]: Code snippets should be preceded by a **“colon” (:)**
```suggestion
    If you want to use tensors whose dimension is higher than 4, use the following function:
```

**Review by omar-srbd — COMMENTED** [review]
Please check the suggestion, thanks.

**Review by omar-srbd — APPROVED** [review]
@niley7464, thanks for incorporating the changes.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**